### PR TITLE
[WIP] Docs: typos + minor changes

### PR DESCRIPTION
**Pull request by kanishk16**
FIX: typo and links changed to hyperlinks

*Hello, I have made some minor changes of converting the links into hyperlinks and corrected a typo while going through the documentation for the installation of GeNN. Just wanted to confirm if this doesn't introduce any inconsistencies that I'm unaware of and undo them if it does.*

### CODE REVIEW

File: doxygen/01_Installation.dox [hunk 1]
```diff
@@ -10,33 +10,33 @@ Point your browser to \a https://github.com/genn-team/genn/releases
...
 
 This will clone the entire repository, including all open branches.
 By default git will check out the master branch which contains the
 source version upon which the next release will be based. There are other 
 branches in the repository that are used for specific development 
 purposes and are opened and closed without warning.
 
-As an alternative to using git you can also download the full content of
-GeNN sources clicking on the "Download ZIP" button on the bottom right
-of the GeNN Github page (\a https://github.com/genn-team/genn).
```
Comment (tnowotny) [inline doxygen/01_Installation.dox]: When processed through doxygen these links work fine as is - so no need to make explicit hyperlinks.

**Comment by kanishk16** [comment]
Done...I'll be adding other doc fixes also (if any) in this PR only

File: doxygen/02_Quickstart.dox [hunk 2]
```diff
@@ -14,20 +14,20 @@
 //----------------------------------------------------------------------------
 /*! \page Quickstart Quickstart 
 
-GeNN is based on the idea of code generation for the involved GPU or
-CPU simulation code for neuronal network models but leaves a lot of
-freedom how to use the generated code in the final
+GeNN is based on the idea of code generation, not only generating GPU or
+CPU simulation code for neuronal network models but also leaving a lot of
+freedom on how to use the generated code in the final
```
[the remark this answers is not in the document]
Comment (kanishk16) [reply doxygen/02_Quickstart.dox]: Yess, this is simple and with optimum clarity.

File: userproject/Model_Schmuker_2014_classifier_project/README.txt [hunk 3]
```diff
@@ -1,66 +1,62 @@
-Author: Alan Diamond, University of Sussex, 2014
```
Comment (kanishk16) [inline userproject/Model_Schmuker_2014_classifier_project/README.txt]: Do we need to add the author as it creates a bit of confusion...also it has already been cited in the model files

File: userproject/Model_Schmuker_2014_classifier_project/README.txt [hunk 4]
```diff
@@ -1,66 +1,62 @@
...
+A neuromorphic network for generic multivariate data classification
+===================================================================
 
-This project recreates using GeNN the spiking classifier design used in the paper
+This project recreates the spiking classifier using GeNN as proposed in the paper by
```
Comment (kanishk16) [inline userproject/Model_Schmuker_2014_classifier_project/README.txt]: Is this description as 'spiking classifier' articulate?

File: doxygen/01_Installation.dox [hunk 5]
```diff
@@ -29,14 +29,14 @@ source version upon which the next release will be based. There are other
 branches in the repository that are used for specific development 
 purposes and are opened and closed without warning.
 
-As an alternative to using git you can also download the full content of
-GeNN sources clicking on the "Download ZIP" button on the bottom right
+As an alternative to using git is to download the full content of
```
Comment (neworderofjamie) [inline doxygen/01_Installation.dox:32]: This no longer makes sense - you need to remove the "As"

File: doxygen/01_Installation.dox [hunk 6]
```diff
@@ -65,22 +65,19 @@ repository.
      For Windows, download Microsoft Visual Studio Community Edition from
      \a https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx.
      When installing Visual Studio, one should select the 'Desktop 
-    development with C++' configuration' and the 'Windows 8.1 SDK' and 'Windows 
+    development with C++ configuration' and the 'Windows 8.1 SDK' and 'Windows
```
Comment (neworderofjamie) [inline doxygen/01_Installation.dox:68]: the quotes were wrong before but they are now wrong in a different way - it should read

>  'Desktop development with C++' configuration

File: userproject/Model_Schmuker_2014_classifier_project/README.txt (hunk 4 above)
Comment (neworderofjamie) [reply userproject/Model_Schmuker_2014_classifier_project/README.txt:4]: I think

> This project recreates the spiking classifier proposed in the paper by Michael Schmuker, Thomas Pfeil and Martin Paul Nawrota using GeNN

would be better

Comment (kanishk16) [reply userproject/Model_Schmuker_2014_classifier_project/README.txt:4]: Yes, definitely.

File: doxygen/01_Installation.dox (hunk 6 above)
Comment (kanishk16) [reply doxygen/01_Installation.dox:68]: Corrected and updated these changes.

**Comment by kanishk16** [comment]
Sorry for the late response. I have made the suggested changes and would like to know if I possibly missed anything :)

**Pull request merged**